helm/etcd-backup-operator/values.yaml

```diff
@@ -45,3 +45,5 @@ installation: ""
 
 verticalPodAutoscaler:
   enabled: true
+# priorityClassName used by the pod.
+priorityClassName: ""
```

Comment:
I'd set a value as default so we solve the scheduling issue on all MCs in one step without requiring any change in the config repo

Reply:
wouldn't it be the same if you keep it configurable as you did, but specify a default in the chart's values?
You wouldn't need a (single, true) PR in `config` but you'd still be able to override the value.

anyway your choice